src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFromMarkdownCommand.cs

```diff
@@ -57,23 +57,24 @@ public class ConvertFromMarkdownCommand : PSCmdlet
         private const string LiteralPathParameterSet = "LiteralParamSet";
         private const string InputObjParamSet = "InputObjParamSet";
         private MarkdownConversionType conversionType = MarkdownConversionType.HTML;
-        private MarkdownOptionInfo mdOption = null;
+        private PSMarkdownOptionInfo mdOption = null;
 
         /// <summary>
-        /// Read the MarkdownOptionInfo set in SessionState.
+        /// Read the PSMarkdownOptionInfo set in SessionState.
         /// </summary>
         protected override void BeginProcessing()
         {
-            mdOption = SessionState.PSVariable.GetValue("MarkdownOptionInfo", new MarkdownOptionInfo()) as MarkdownOptionInfo;
+            mdOption = SessionState.PSVariable.GetValue("PSMarkdownOptionInfo", new PSMarkdownOptionInfo()) as PSMarkdownOptionInfo;
```

Comment:
Same comment as in https://github.com/PowerShell/PowerShell/pull/7329#discussion_r204597710

Reply:
Fixed.